easybuild/easyconfigs/l/LASTZ/LASTZ-1.02.00-foss-2016a.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'ConfigureMake'
+
+name = 'LASTZ'
+version = '1.02.00'
+
+homepage = 'http://www.bx.psu.edu/~rsharris/lastz/'
+description = """ LASTZ is a program for aligning DNA sequences, a pairwise aligner. Originally designed to handle
+ sequences the size of human chromosomes and from different species, it is also useful for sequences produced by NGS 
+ sequencing technologies such as Roche 454.
+"""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['http://www.bx.psu.edu/miller_lab/dist/']
+
+patches = ['LASTZ-Makefile.patch']
```

Comment:
@RvDijk this patch is missing?

also, please rename it to include the version (since it may break for future versions), something like: `LASTZ-1.02.00_Makefile.patch`